app/models/manageiq/providers/ansible_tower/provider.rb

```diff
@@ -60,6 +62,18 @@ def url=(new_url)
     default_endpoint.url = uri.to_s
   end
 
+  def self.seed
```

Comment:
@durandom sorry, didn't see this comment.  Maybe?  This is just a placeholder for now.  I'll be integrating with NickC's work and we'll be addressing how we find/seed the default tower per zone.